mariner/config.py

```diff
@@ -35,6 +35,24 @@ def get_files_directory() -> Path:
     return Path(str(config.get("files_directory", "/mnt/usb_share")))
 
 
+def get_file_sort_order() -> str:
```

Comment:
instead of returning a `str` here, can you return an enum? the main advantage is that it will be clear what are all the possible values that can be returned by this function - as opposed with a string where pretty much anything could be here

Reply:
No idea how to, sorry.